Tasks/VsTest/vstest.ts

```diff
@@ -130,6 +136,42 @@ function getTestAssemblies(): Set<string> {
     return new Set(testAssemblyFiles);
 }
 
+function addVstestDiagOption(argsArray: string[]) {
+    let sysDebug = tl.getVariable("System.Debug");
+    if (sysDebug !== undefined && sysDebug.toLowerCase() === "true") {
```

Comment:
refactor this code. Why so many if else conditions? function is too big to accept